test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -81,11 +81,13 @@ func StatefulSetIsReadyAfterScaleDown(mdb *mdbv1.MongoDBCommunity) func(t *testi
 // reaches the running state.
 func statefulSetIsReady(mdb *mdbv1.MongoDBCommunity, opts ...wait.Configuration) func(t *testing.T) {
 	return func(t *testing.T) {
+		start := time.Now()
 		err := wait.ForStatefulSetToBeReady(t, mdb, opts...)
 		if err != nil {
 			t.Fatal(err)
 		}
-		t.Logf("StatefulSet %s/%s is ready!", mdb.Namespace, mdb.Name)
+		elapsed := time.Since(start)
+		t.Logf("StatefulSet %s/%s is ready! It took %d seconds", mdb.Namespace, mdb.Name, elapsed)
```

Comment:
If we have timeout panics (overall runner failed) we will still have the time it took in the logs